### PR TITLE
FUSETOOLS2-2313: Add UI tests for Display variable values with Exchange scope during d…

### PR DESCRIPTION
**Pull request by pospisilf**
…ebugging

### CODE REVIEW

File: src/ui-test/resources/variablesTest.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import org.apache.camel.builder.RouteBuilder;
+
+public class variablesTest extends RouteBuilder {
```
Comment (apupier) [inline src/ui-test/resources/variablesTest.java:3]:
```suggestion
public class VariablesTest extends RouteBuilder {
```

File: src/ui-test/resources/variablesTest.java
Comment (apupier) [inline src/ui-test/resources/variablesTest.java:1]: filename to change to foollow Java convention: VariablesTest.java

**Review by apupier — CHANGES_REQUESTED** [review]
I think it is worthy mentioning in the commit message that it is the tests which are added by this commit

**Comment by djelinek** [comment]
@pospisilf the new tests are failing due to last changes on PR

**Comment by pospisilf** [comment]
> @pospisilf the new tests are failing due to last changes on PR

Fixed. Thanks for noticing.

**Review by djelinek — APPROVED** [review]

**Comment by djelinek** [comment]
> I think it is worthy mentioning in the commit message that it is the tests which are added by this commit

@apupier which commit do you mean?

**Comment by pospisilf** [comment]
> > I think it is worthy mentioning in the commit message that it is the tests which are added by this commit
> 
> @apupier which commit do you mean?

already solved in past by me, sorry

**Review by apupier — APPROVED** [review]

**Pull request merged**